packages/core/admin/ee/server/validation/review-workflows.js

```diff
@@ -21,7 +21,13 @@ const validateUpdateStageOnEntity = yup
   })
   .required();
 
+const validateWorkflowCreateSchema = yup.object().shape({
+  name: yup.string().max(255).required(),
+  stages: yup.array().of(stageObject),
```

Comment:
If this is required could we use `array.min(limit: number, message?: string)` from yup?